src/Entity/CommitteeElection.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="CommitteeElectionRepository")
+ */
+class CommitteeElection
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var Committee
+     *
+     * @ORM\OneToOne(targetEntity="Committee", inversedBy="committeeElection")
+     * @ORM\JoinColumn(nullable=false)
```

Comment:
you should add `onDelete="CASCADE"` to explain cascades to the database.

(if we remove a comittee => all related elections should be removed, instead of raising an error because of a foreign key referencing this committee)